e2e/cypress/support/api_commands.js

```diff
@@ -494,6 +494,24 @@ Cypress.Commands.add('apiSaveThemePreference', (value = JSON.stringify(theme.def
     });
 });
 
+/**
+ * Saves the preference over link preview
+ * This API assume that the user is logged in and has cookie to access
+ * @param {boolean} value - theme object.  Will pass default value if none is provided.
+ */
+Cypress.Commands.add('apiSaveLinkPreviewPreference', (showExpanded = false) => {
```

Comment:
Let's just accept string value of 'true' or 'false'
```suggestion
Cypress.Commands.add('apiSaveLinkPreviewPreference', (show = 'true') => {
```